ubuntu/xenial/Dockerfile

```diff
@@ -40,5 +40,12 @@ RUN apt-get update && apt-get install -y --force-yes \
     alien \
     dh-systemd
 
+# Install gcc-8
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test -y &&  \
```

Comment:
This change is too Tarantool-specific. If you want to do these changes at this repository, you should update all Ubuntu images where GCC version < 8 for consistency. Otherwise, it looks very strange to update Ubuntu Xenial only. 

I suggest creating own Docker image on the base of PackPack Ubuntu Xenial image and put it in the tarantool/testing repo on Docker Hub. In this case, you will be able to adjust the image as you need.